cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -5,7 +5,7 @@ namespace: default
 labels:
   k8s-app: skydns
 desiredState:
-  replicas: {DNS_REPLICAS}
+  replicas: {{ pillar['dns_replicas'] }}
```

Comment:
Hm. Are you talking about `cluster/addons/dns/README.md`? Those
instructions don't look right ... they don't include any instructions on
how you'd bonk salt to update the minions for the `DNS_SERVER_IP`. They
look pre-ENABLE_CLUSTER_DNS, pre-salt config, at the very least. At the
moment, the configuration seems pretty static once you create the cluster,
unless you know the right incantations.

Given that, would you mind a separate pull for cleanup on the docs?

On Tue, Jan 20, 2015 at 9:26 AM, Tim Hockin <notifications@github.com>
wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601#discussion-diff-23238511>
> :
>
> > @@ -5,7 +5,7 @@ namespace: default
> >  labels:
> >    k8s-app: skydns
> >  desiredState:
> > -  replicas: {DNS_REPLICAS}
> > +  replicas: {{ pillar['dns_replicas'] }}
>
> There is a doc that explain to mortal users how DNS works. Needs updates?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601/files#r23238511>
> .
>